emeritus-tech/landing-pages-libraries
# [CPT-287] Configure and implement storybook for Landings documentation

🎟 [**Jira Ticket**](https://emeritus.atlassian.net/browseCPT-287)



### 🚢 Deployment Instructions

* N/A

### ✅ QA instructions

Just run `npm run storybook` and:
* Check that storybook load and works properly
* Check that styles are loading and are the corrects ones for the landing
* Check that all middle sections are documented and are the corrects ones for the landing

### ⚠️ Risks

* Low

### 🖼 Previews

<img width="1675" alt="Screen Shot 2020-05-22 at 10 19 39 AM" src="https://user-images.githubusercontent.com/251192/82671750-c2858d00-9c15-11ea-9f4c-68e13978b244.png">

### 📝 Notes

* Berkeley and INSEAD stories are missing since those landings weren't merge yet.
* Added a specific `_story.scss` file in overrides folder to handle specific styles related to the storybook presentation.
